src/UnkObjects/unk5769E0.hpp

```diff
@@ -73,4 +73,4 @@ class unk_5769E0 {
 #define glo_6C5254 (MC2_GLOBAL<bool>(0x006C5254))
 #define global_primary_unk5769E0 (MC2_GLOBAL<unk_5769E0 *>(0x006C5258)) // glo_6C5258
 
-#define loc_6C524C (MC2_POINTER<void>(0x006C524C))
+#define loc_6C524C (MC2_POINTER<int32_t>(0x006C524C))
```

Comment:
In general, `MC2_POINTER<void>` is a cheat I use when I don't know what the type is that I'm referring to. Whenever you figure out the type for this sort of thing, you should be changing it to `MC2_GLOBAL`, changing the `loc_` to `glo_` and replacing the instances of `loc_X` to `&glo_X`.